daemons/dss-index/app.py

```diff
@@ -11,4 +13,5 @@
 
 @app.s3_event_handler(bucket=s3_bucket, events=["s3:ObjectCreated:*"])
 def dispatch_indexer_event(event, context) -> None:
+    app.log.setLevel(logging.DEBUG)
```

Comment:
NB this doesn't propagate to other loggers, so we won't get spam from connexion/urllib3 and such.